solr/app.py

```diff
@@ -17,10 +19,37 @@ def create_app():
     app.url_map.strict_slashes = False
     Consul(app)  # load_config expects consul to be registered
     load_config(app)
+    if config:
+        app.config.update(config)
+        
+    db.init_app(app)
     logging.config.dictConfig(
         app.config['SOLR_SERVICE_LOGGING']
     )
     
+    ## pysqlite driver breaks transactions, we have to apply some hacks as per
```

Comment:
If I understand correctly, this block of code is so that you can use `sqlite:///` in-memory db to run the tests against, but the live service would be in a postgres db? If this is the only reason, I'd suggest we use TestingPostgres instead (one example, but not perfect, is here: https://github.com/adsabs/biblib-service/blob/82e9ec1ed11fe0e20b75946b5e3bfe521710d124/biblib/tests/unit_tests/test_manage.py#L43).

It's a little more overhead in that you need postgres, but this is easy to add in `.travis.yml`. Otherwise, I think people may think this is core to `create_app` and could cause misunderstanding.

Thoughts?